src/shortcodes/image-position-with-text.js

```diff
@@ -2,15 +2,14 @@ const markdownFilter = require('../filters/markdown.js');
 
 module.exports = (content, image, alt, imagePosition, scale, maxHeight, verticalAlignment) => {
 	content = content.trim();
-	maxHeight = maxHeight.trim() === 'auto' ? 'auto' : maxHeight + 'px';
+	let output = '';
 
-	let output;
-	output = `<style>:root {--image-position-max-height: ${maxHeight}; --image-position-width: ${scale};}</style>`;
 	if (content === '') {
-		output += `<img src="${image}" alt="${alt}" class="image-position--${imagePosition} image-position--image-height" />`;
+		output += `<p><img src="${image}" alt="${alt}" class="image-position--${imagePosition} image-position--image-height  image-and-text__scale--${scale} image-and-text__max-height--${maxHeight}" /></p>`;
 	} else {
-		imagePosition = imagePosition.trim() === 'center' ? 'left' : imagePosition;
-		output += `<div class="image-and-text image-and-text--image-${imagePosition} image-and-text--vertical-${verticalAlignment}"><figure class="image-and-text__media"><img src="${image}" alt="${alt}" class="image-position--image-height" /></figure><div class="image-and-text__content">${markdownFilter(content)}</div></div>`;
+		scale = scale === '100' ? '75' : scale; // Reduce to 75% width to allow content to be shown beside it.
```

Comment:
It does appear that there are different rules at larger breakpoints, see: https://github.com/jhung/idrc/blob/92f36cc6045e625c58cf4ba923767e4cd828bdb2/src/assets/styles/vendors/_cms-editor-components.scss#L136-144

Reply:
oh, I meant classes for displaying text to left and right of the image (there is no .image-and-text--image-left--100 or .image-and-text--image-right--100). This is because to display text next to the image, text need some spaces next to it. I think we shouldn't have 100 option at all for these cases.

Reply:
@chosww and @greatislander I was rethinking this situation where the user has specified 100% width for an image and has also specified some text / content. In this situation it might be less confusing to have the image 100% width in a row above the content.

Thoughts?

Reply:
I think that makes sense @jhung, and that would be clearly distinguishable with 75% option.